README.md

```diff
@@ -47,6 +47,12 @@ const user = { email: randEmail(), name: randFullName() };
 const emails = randEmail({ length: 10 });
 ```
 
+You can specify the length of elements you want to generate. Below is an example of generating 10 emails with length equal or smaller than 20 characters.
+
+```ts
+const emails = randEmail({ length: 10, maxCharCount: 20 });
```

Comment:
The returned size can differ from the `length` value, right? I wonder if this is the desired result.

Reply:
Isn't that length is just quantity of items and maxChartCount is length of that items?

Reply:
Yes, but what happens when you request 20 items and only 10 pass the maxCharCount predicate?

Reply:
Then there will be some duplicate values in return

Reply:
Can you please add a test for this case?

Reply:
https://github.com/ngneat/falso/pull/316/files#diff-b93ef17757be75c3c90f47b1c860c47f9eaa31865b28df81874fe2043686cfe5R12

There is such a test, should I increase the length and decrease maxCharCount?

Reply:
Yes, please. In a new spec.

Reply:
@NetanelBasal done